scripts/check_doc_ipfs_hashes.py

```diff
@@ -49,6 +49,9 @@
 PACKAGE_TABLE_REGEX = rf"\| {PACKAGE_TYPE_REGEX}\/{VENDOR_REGEX}\/{PACKAGE_REGEX}\/{VERSION_REGEX}(\s|\|)*(?P<hash>{IPFS_HASH_REGEX})\s*\|"
 
 ROOT_DIR = Path(__file__).parent.parent
+# We need to skip the hash for `valory/oracle_hardhat:0.1.0:bafybeie553shfmnds6v7defynjv5kmjkqf2aygj345jbbcssevtnkbodbe`
+# because the demo is not in this repo, but the docs are
+HASH_SKIPS = ("bafybeie553shfmnds6v7defynjv5kmjkqf2aygj345jbbcssevtnkbodbe",)
```

Comment:
There is an alternative about this, I suggested sometime ago: Don't write hashes in the docs, and write some macro which automatically checks the last Tagged version for the correct hash and populates it at compile time. E.g.,
autonomy fetch {% get_hash(valory/oracle) %}  etc..

Reply:
(in this case, it should be required to indicate the repo as well)